ruleguard/parser.go

```diff
@@ -234,6 +234,9 @@ func (p *rulesParser) parseRuleGroup(f *ast.FuncDecl) error {
 	defer p.itab.LeaveScope()
 
 	for _, stmt := range f.Body.List {
+		if _, ok := stmt.(*ast.DeclStmt); ok {
```

Comment:
Should probably check for decl stmt token for `token.CONST`. Otherwise, it would also permit vars and type defs.